src/networks.json

```diff
@@ -1453,7 +1453,7 @@
     "chainId": 22040,
     "network": "testnet",
     "testnet": true,
-    "multicall": "0x18f7BFad831D8ED9E16C4cD50D6041dD453Ca1EF",
+    "multicall": "0xFCE5b5ca100e58E5635B15b79F0163be4CbD258b",
```

Comment:
Hi, @SvineruS Why do you want to change this?

Reply:
Our testnet was redeployed on same chain id; The old contact no longer exists

Reply:
This multicall contract is not verified and published here https://testnet.airdao.io/explorer/address/0xFCE5b5ca100e58E5635B15b79F0163be4CbD258b/contract/verify/

Reply:
verified now